### PR TITLE
Feature/stats horizontal

### PR DESCRIPTION
**Pull request by ramneet-persistent**
Stats horizontal chart

### CODE REVIEW

File: dashboards-observability/public/components/visualizations/charts/stats/stats.tsx [hunk 1]
```diff
@@ -0,0 +1,686 @@
...
+        ...chartLayout,
+        annotations: chartLayout.annotations.concat(
+          orientation === DefaultOrientation || metricsLength === 1
+            ? createAnnotationAutoModeVertical(annotationOption)
+            : createAnnotationsAutoModeHorizontal(annotationOption)
+        ),
+        [`xaxis${metricIndex > 0 ? metricIndex + 1 : ''}`]: {
+          visible: false,
+          showgrid: false,
+          anchor: `y${metricIndex > 0 ? metricIndex + 1 : ''}`,
```
Comment (spattnaik) [inline dashboards-observability/public/components/visualizations/charts/stats/stats.tsx]: `metricIndex > 0 ? metricIndex + 1 : ''` You can put this in a variable and use it everywhere. I see you have been doing this 4 times. We can have just one comparison, if we save it in a variable.

File: dashboards-observability/public/components/visualizations/charts/stats/stats.tsx [hunk 2]
```diff
@@ -0,0 +1,686 @@
...
+                valueColor: '',
+              })
+            : createAnnotationTextModeHorizontal({
+                label: metric.label,
+                value: getMetricValue(metric.label),
+                index: metricIndex,
+                valueColor: '',
+              })
+        ),
+        [`yaxis${metricIndex > 0 ? metricIndex + 1 : ''}`]: {
```
Comment (spattnaik) [inline dashboards-observability/public/components/visualizations/charts/stats/stats.tsx:485]: Similar to above comment.

**Review by spattnaik — APPROVED** [review]
Few minor comments. Please fix before moving to observability repo

**Comment by ramneet-persistent** [comment]
raised separate internal PR with 2 way sync [internal PR 12#](https://github.com/ramneet-persistent/observability/pull/12) 
so closing this one

**Pull request closed without merging**